core/domain/rights_manager.py

```diff
@@ -184,18 +214,45 @@ def _save_activity_rights(
 # same as the activity id).
 # TODO(msl): get rid of inline imports by refactoring code
 def _update_exploration_summary(activity_rights):
+    """Updates the exploration summary field for an activity.
```

Comment:
Changed it slightly for following the conventions. 

"Updates the exploration summary for the given activity(associated with the given rights object).
The ID of rights object is the same as the ID of associated activity."